src/main/java/cn/hashdata/bireme/Table.java

```diff
@@ -57,14 +57,15 @@ public Table(String tableName, Map<String, List<String>> tableMap, Connection co
     ResultSetMetaData rsMetaData = null;
 
     try {
-      List<String> mapList = tableMap.get(tableName);
+      String[] strArray = tableName.split("\\.");
+      List<String> mapList = tableMap.get(strArray[1]);
```

Comment:
The table name should include schema info. So you should modify the input parameters of getPrimaryKeys

```
String tableName = resultSet.getString("TABLE_SCHEM") + '.' + resultSet.getString("TABLE_NAME");
```